spec/lib/user_segments_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+describe UserSegments do
+
+  describe "#all_users" do
+    it "returns all active users with newsletter enabled" do
+      active_user1 = create(:user, newsletter: true)
+      active_user2 = create(:user, newsletter: true)
+      active_user3 = create(:user, newsletter: false)
+      erased_user  = create(:user, erased_at: Time.current)
+
+      expect(UserSegments.all_users).to include active_user1
```

Comment:
I think its a good practice to use `described_class` instead of the class name like: